views/features/steps/common.py

```diff
@@ -0,0 +1,7 @@
+import nose
+from behave import then
+
+
+@then('a success status code (200) is returned')
+def step_impl_success_returned(context):
```

Comment:
`step_impl_success_returned` -> `success_returned`. Also it would be good to rename the file to something so it's more cohesive and less likely to be a dumping ground for things with no home. Maybe response_status.py?